src/main/java/newbank/server/Customer.java

```diff
@@ -40,4 +40,15 @@ public String getUsername() {
   public String getPassword() {
     return password;
   }
+  
+  @Override
+  public String toString() {
+	  StringBuilder sb = new StringBuilder();
+	  sb.append(this.getClass().getName() + "[");
```

Comment:
No newlines in this output? Does this look readable in the current form?

Reply:
The to_string does not add newlines. The client will add newlines if needed.